aws_xray_sdk/ext/django/middleware.py

```diff
@@ -46,12 +51,15 @@ def __call__(self, request):
             sampling_req=sampling_req,
         )
 
-        segment = xray_recorder.begin_segment(
-            name=name,
-            traceid=xray_header.root,
-            parent_id=xray_header.parent,
-            sampling=sampling_decision,
-        )
+        if self.in_lambda:
+            segment = xray_recorder.begin_subsegment(name)
```

Comment:
Trace header information, including the sampling decision, is propagated through to the Facade Segment.